solidity/random-beacon/test/RandomBeacon.GroupCreation.test.ts

```diff
@@ -4,6 +4,7 @@ import { ethers, waffle, helpers, getUnnamedAccounts } from "hardhat"
 import { expect } from "chai"
 import type { BigNumber, ContractTransaction, Signer } from "ethers"
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { describe } from "mocha"
```

Comment:
🤔 

Is it needeed?

Reply:
https://github.com/keep-network/keep-core/pull/2735/commits/7cf5c0e2a88586386d58fbc84d1c50b5f87d3919